__mocks__/@tanstack/react-virtual.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
If we use a global mock, how do we ensure that unit tests of our consumers will not fail? Can we not use such ultimate tool?

Reply:
Updated

Reply:
What is updated here? This mock still applies to all of our tests. We never test against the actual virtual instance in JSDOM.

Do you think it is okay? What if something fails in our live consumers?

Reply:
How do we release this to live, if it has required upgrade steps to our consumers?